### PR TITLE
Shared library support

### PR DESCRIPTION
**Pull request by wirx6**

### CODE REVIEW

File: QDiscord/src/src.pro [hunk 1]
```diff
@@ -1,52 +1,59 @@
-QT       += network websockets
```
Comment (george99g) [inline QDiscord/src/src.pro]: Wall of pink. This repository uses LF instead of CRLF.

Comment (wirx6) [inline QDiscord/src/src.pro]: fixed

**Comment by george99g** [comment]
This PR is turning into two features, now.

**Comment by george99g** [comment]
Implemented in 2a2e91f9beeda1f967c040f756e2bef03599ea95

**Comment by george99g** [comment]
Also, another thing to note: `qdiscordutilities.hpp` serves the function of `qdiscord_global.hpp`, so `qdiscord_global.hpp` is kinda unnecessary.

**Pull request closed without merging**